utils/tests/test_hosts.py

```diff
@@ -0,0 +1,162 @@
+# -*- coding: utf-8 -*-
+""" Tests utils/hosts.py """
+import re
+import pytest
+import os
+from unittest import mock
+import logging
+
+from locust.env import Environment
+
+from utils.hosts import MilMoveHostMixin, MilMoveDomain, MilMoveEnv, clean_milmove_host_users
+from utils.base import ImplementationError
+
+
+class TestMilMoveDomain:
+    """ Tests the MilMoveDomain class and its methods. """
+
+    def test_host_name(self):
+        assert (
+            MilMoveDomain.match(MilMoveDomain.PRIME).host_name("local", True, 1111, "http") == "http://primelocal:1111"
+        )
+        assert MilMoveDomain.match(MilMoveDomain.PRIME).host_name("exp") == "https://prime.exp.move.mil"
+
+        with pytest.raises(ImplementationError):
+            MilMoveDomain.match(MilMoveDomain.PRIME).host_name("local", True, 11111)
+
+        with pytest.raises(ImplementationError):
+            MilMoveDomain.match(MilMoveDomain.PRIME).host_name("test")
+
+
+class TestMilMoveHostMixin:
+    """ Tests the MilMoveHostMixin class and its methods. """
+
+    @classmethod
+    def setup_class(cls):
+        """ Define and initialize classes to be tested """
+
+        class HostLocal(MilMoveHostMixin):
```

Comment:
I might suggest renaming this to something like `TestHostUser` instead of `HostLocal`. It's not _always_ going to be local during the tests, plus this obfuscates the fact that this is meant to be used like an `HttpUser` class. BUT this is definitely up to you, I'm fine with it if you keep it as-is too